client/src/components/TextArea/Text.js

```diff
@@ -1,12 +0,0 @@
-import React from 'react'
-import './Text.css'
-
-
-const Text = (props) => <div>
-<p className={props.pClassList}>Is there anything else you want us to know?</p>
-<textarea className={props.textAreaClassList}>
-Tell us about your dream dress! Do you have any Modern Trousseau favorites?
-</textarea></div>
-
-
-export default Text
```

Comment:
Did you mean to delete this file? If you merge this branch into develop and have deleted this file it will remove it from the delete branch.

Reply:
This was Dilmourads.